LEGO1/lego/legoomni/src/race/jetskirace.cpp

```diff
@@ -1,28 +1,168 @@
 #include "jetskirace.h"
 
+#include "actions/jetrace_actions.h"
+#include "actions/jukebox_actions.h"
+#include "dunebuggy.h"
+#include "isle.h"
+#include "legoanimationmanager.h"
+#include "legocontrolmanager.h"
+#include "legohideanimpresenter.h"
+#include "legomain.h"
+#include "legopathstruct.h"
+#include "legoracespecial.h"
+#include "legoutils.h"
+#include "misc.h"
+#include "mxactionnotificationparam.h"
+#include "mxbackgroundaudiomanager.h"
+#include "mxmisc.h"
+#include "mxstillpresenter.h"
+#include "mxtransitionmanager.h"
+#include "mxvariabletable.h"
+#include "scripts.h"
+
+// Defined in legopathstruct.cpp
+extern MxBool g_unk0x100f119c;
+
+// Defined in jetski.cpp
+extern const char* g_varJSFRNTY5;
+extern const char* g_varJSWNSHY5;
+
+// Defined in legoracespecial.cpp
+extern const char* g_raceState;
+extern const char* g_racing;
+
+// Defined in legopathactor.cpp
+extern const char* g_strHIT_WALL_SOUND;
```

Comment:
Not sure if we want to stick to this pattern or make these available in the header file. I've seen both.

Reply:
I don't have a strong opinion on either, so I think we can leave it like this for now. I think it's better practice to put them in respective header files but we can refactor this later.